tvb/simulator/plot/tools.py

```diff
@@ -520,8 +681,204 @@ def channel_formatter(x, pos=None):
+    title("Temporal average -- State variable V")
+
+    figure(2)
+    plot(tavg_time[t_interval], TAVG[t_interval, 1, :, 0], 'b', alpha=0.05)
+    plot(tavg_time[t_interval], TAVG[t_interval, 1, :, 0].mean(axis=1), 'b', linewidth=3)
+    title("Temporal average -- State variable W")
+
+    figure(3)
+    plot(tavg_time[t_interval], TAVG[t_interval, 2, :, 0], 'r', alpha=0.05)
+    plot(tavg_time[t_interval], TAVG[t_interval, 2, :, 0].mean(axis=1), 'r', linewidth=3)
+    title("Temporal average -- State variable Z")
+    xlabel('time [ms]', fontsize=24)
+def sim_sum_eeg():
+    """
+    Cortical surface with subcortical regions,
+    sEEG, EEG & MEG, using a stochastic integration.
+    """
+    figure()
+
+    for i, mon in enumerate((eeg, meg, seeg)):
+        subplot(3, 1, i + 1)
+        time, data = mon
+        plot(time, data[:, 0, :, 0], 'k', alpha=0.1)
+        ylabel(['EEG', 'MEG', 'sEEG'][i])
+
+    tight_layout()
+
+def reg_red_wong_wang():
+    """
+    perform a simulation with the reduced Wong-Wang model,
+    using the default connectivity.
+    :return:
+    """
+    figure('Position', [500 500 1000 400])
+    subplot(121, imagesc(np2m(conn.weights)), colorbar, title('Weights'))
+    subplot(122, imagesc(np2m(conn.tract_lengths)), colorbar)
+    title('Tract Lengths (mm)')
 
-def plot_fast_kde(x, y, kern_nx = None, kern_ny = None, gridsize=(500, 500), 
+def matlab_two_ep_sim():
+    """
+    erform a simulation with two Epileptors.
+    :return:
+    """"""
+    figure()
+
+    subplot(311)
+    plot(time, squeeze(signal(1, :, 1, :)), 'k')
+    ylabel('x2(t - x1(t)')
+    set(gca, 'XTickLabel', {})
+
+    title('Two Epileptors')
+
+    # plot high-pass filtered LFP
+    subplot(312)
+    [b, a] = buffer(3, 2/2000*5.0, 'high')
+    hpf = filter(b, a, squeeze(signal(1,:, 1,:)
+    plot(time, hpf(:, 1), 'k')
+    plot(time, hpf(:, 2), 'k')
+    set(gca, 'XTickLabel', {})
+    ylabel('HPF LFP')
+
+    subplot(313)
+    plot(time, squeeze(signal(1,:, 2,:)), 'k')
+    ylabel('Z(t)')
+    xlabel('Time (ms)')
+
+def region_simulate():
+    """
+    presents the basic anatomy of a region simulation using
+    The Virtual Brain's (TVB's) scripting interface.
+    :return:
+    """
+    # Model
+    oscilator = models.Generic2dOscillator()
+    # Connectivity
+    white_matter = connectivity.Connectivity(load_default=True)
+    white_matter.speed = numpy.array([4.0])
+    # Coupling
+    white_matter_coupling = coupling.Linear(a=0.0154)
+    # Integrator
+    heunint = integrators.HeunDeterministic(dt=2 ** -6)
+    # monitors
+    # Initialise some Monitors with period in physical time
+    mon_raw = monitors.Raw()
+    mon_tavg = monitors.TemporalAverage(period=2 ** -2)
+    # Bundle them
+    what_to_watch = (mon_raw, mon_tavg)
+    # simulator
+    # Initialise a Simulator -- Model, Connectivity, Integrator, and Monitors.
+    sim = simulator.Simulator(model=oscilator, connectivity=white_matter,
```

Comment:
Again, running simulations should not be in the plotting library

Reply:
Ok got it removed